tests/settings.py

```diff
@@ -21,7 +21,7 @@
 MIDDLEWARE_CLASSES = (
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+    # 'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
Another option would be to use the [csrf_exempt](https://docs.djangoproject.com/en/1.10/ref/csrf/#django.views.decorators.csrf.csrf_exempt) decorator on premis-event-service's views, rather than requiring that the middleware be disabled for the whole project, which in turn would require the user to go back and place the csrf_protect decorator on each of their own views manually in order to get the protection it offers. I haven't looked to see where the conflict occurs, so this may not work, but I thought I'd throw it out there.